test/gtest/graphapi_gtest_common.hpp

```diff
@@ -98,8 +98,8 @@ class GTestDescriptorVectorAttribute : public GTestDescriptorAttribute
                                    int64_t invalidCount,
                                    std::initializer_list<ValueType> values)
         : mValues(values),
-          mInvalidTypeValues(std::max(1ul, values.size())),
-          mInvalidCountValues(std::max(1l, invalidCount),
+          mInvalidTypeValues(std::max(static_cast<decltype(values.size())>(1), values.size())),
+          mInvalidCountValues(std::max(static_cast<decltype(invalidCount)>(1), invalidCount),
```

Comment:
@scerzh Yes, we should never use UL and L in literals. Static casts or ULL/LL are fine.

Reply:
@atamazov what about L or F in floating point literals?

Reply:
@scerzh Word "integer" added. Good catch, thanks))